frontend/src/recoil/instrument.ts

```diff
@@ -0,0 +1,8 @@
+import { Instrument } from "common/dist/ensembles/Note";
+import { atom } from "recoil";
+
+/** A value between 0 and 32 */
```

Comment:
I'm not confident this comment is accurate. I mean it's technically true but the comment is misleading. 
```suggestion
/** The user's current instrument, one of {@link Instrument} */
```

If you leave this comment, you can hover over `currentInstrumentAtom` and the comment will appear, with a link to the definition of the `Instrument` enum

Reply:
Yeah forgot to change after copy and pasting